site/styles/_availability-tags.scss

```diff
@@ -62,6 +62,20 @@ s72-availability-label {
     font-size: 12px;
     text-align: center;
   }
+
+  .availability-bg.live {
+    color: var(--body-color, rgb(255, 255, 255));
+    background-color: rgba(var(--body-bg-rgb, 0, 0, 0), 0.7);
+  }
+
+  s72-availability-status {
+    .availability-livenow:before {
```

Comment:
Is the class 'livenow' because we didn't put a dash in before the current status?

Reply:
Nope intentional for consistency see state [here](https://github.com/indiereign/shift72-web/blob/master/src/ui/availability/availability.component.js#L188)

Follows suit with [COMINGSOON](https://github.com/indiereign/shift72-web/blob/master/src/ui/availability/availability.component.js#L166) etc
gets downcased to `class="availability-bg availability-comingsoon"`